net-im/slack-bin/slack-bin-2.1.2.ebuild

```diff
@@ -66,13 +66,15 @@ src_install() {
 	insinto /opt/${MY_PN}
 	doins -r usr/lib/${MY_PN}/.
 	fperms +x /opt/${MY_PN}/${MY_PN}
-	dosym /opt/${MY_PN}/${MY_PN} /usr/bin/${MY_PN}
+	dosym "${ED}"opt/${MY_PN}/${MY_PN} usr/bin/${MY_PN}
```

Comment:
This is 100% wrong. By 'relative symlink' I meant one that starts with appropriate number of `../`s